### ISSUE TITLE
disposition-types?

### ISSUE THREAD

**Issue by jevansAN** [MicrosoftDocs/winrt-api#916]
Would be helpful to list some possible values for the disposition-type string.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 30717193-ecbc-ee8f-b672-9a42f4a6750d
* Version Independent ID: 604e9b21-5150-fed5-3da5-9734950d9233
* Content: [HttpContentDispositionHeaderValue(String) Constructor (Windows.Web.Http.Headers) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.web.http.headers.httpcontentdispositionheadervalue.-ctor#Windows_Web_Http_Headers_HttpContentDispositionHeaderValue__ctor_System_String_)
* Content Source: [https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.web.http.headers.httpcontentdispositionheadervalue.yml&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.web.http.headers.httpcontentdispositionheadervalue.yml&version=GBlive&_a=contents)
* Product: **uwp**
* Git Login: **stevewhims**
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
I've added more info this topic. Links to a list of the valid, standard, values. And a link to a code example in another topic. ~Steve

**Comment by jevansAN** [comment]
Thanks much! Looks like the appropriate Content-Disposition here would be form-data.
Appears I’m not authorized to view the code example. Maybe your team will have something.

Jeff


On Feb 12, 2019, at 12:58 PM, Steven White <notifications@github.com> wrote:

I've added more info this topic. Links to a list of the valid, standard, values. And a link to a code example in another topic. ~Steve

**Comment by stevewhims** [comment]
Ah, yes, I got that link wrong. So, I've just Completed a fix PR, so that should be live either today or tomorrow. The link that I got wrong and just fixed was just to a topic that we've already talked about, though, it's just this one FYI: https://docs.microsoft.com/uwp/api/windows.web.http.headers.httpcontentheadercollection.

**Comment by jevansAN** [comment]
Right! That one I can see. I’ll keep my eyes peeled. 

Jeff


On Feb 12, 2019, at 2:52 PM, Steven White <notifications@github.com> wrote:

Ah, yes, I got that link wrong. So, I've just Completed a fix PR, so that should be live either today or tomorrow. The link that I got wrong and just fixed was just to a topic that we've already talked about, though, it's just this one FYI: https://docs.microsoft.com/uwp/api/windows.web.http.headers.httpcontentheadercollection <https://docs.microsoft.com/uwp/api/windows.web.http.headers.httpcontentheadercollection>.
